terraform/041-id-broker-search-lambda/main.tf

```diff
@@ -29,7 +29,7 @@ resource "aws_iam_role_policy_attachment" "AWSLambdaVPCAccessExecutionRole" {
 resource "aws_lambda_function" "search" {
   s3_bucket        = var.function_bucket_name
   s3_key           = var.function_zip_name
-  source_code_hash = data.http.function-checksum.body
+  source_code_hash = data.http.function-checksum.response_body
```

Comment:
I'm assuming newer version of lambda renamed things?

Reply:
Yes, but not lambda, it's the http data source. Both names are supported in versions 2 and 3 of the provider, but `body` is deprecated.